ecl/hthor/hthorkey.cpp

```diff
@@ -51,7 +51,7 @@ static IKeyIndex *openKeyFile(IDistributedFilePart & keyFile)
             if (thissize != -1)
             {
                 StringBuffer remotePath;
-                rfn.getRemotePath(remotePath);
+                rfn.getPath(remotePath);
```

Comment:
Curious why these changes?

Reply:
I think they are probably correct - getPath() will return RemotePath if it is remote.  I'm not sure why it is calling getRemotePath().  @jakesmith please review all 3 changes like this.

Reply:
yes, I don't know why they were calling getRemotePath before, although harmless, it unnecessarily created a full remote path even when local.
All 4 changes look correct afaics.